openedx/features/wikimedia_features/meta_translations/api/v0/utils.py

```diff
@@ -94,16 +124,31 @@ def get_block_data_from_table(block, wiki_objects):
         block_fields_ids = {}
         wiki_objects = wiki_objects.filter(target_block__block_id=usage_key)
         base_usage_key = ''
+        parsed_status = { 'parsed_block': False }
         for obj in wiki_objects:
             data_type = obj.source_block_data.data_type
             block_fields_ids[data_type] = obj.id
-            block_fields[data_type] = BLOCK_DATA_TYPES_DATA[data_type](obj.translation)
-            base_block_fields[data_type] = BLOCK_DATA_TYPES_DATA[data_type](obj.source_block_data.data)
+            if is_parsed_translation(block.category, data_type):
+                base_decodings = BLOCK_DATA_TYPES_DATA[data_type](obj.source_block_data.parsed_keys)
```

Comment:
can we rename "BLOCK_DATA_TYPES_DATA" to "BLOCK_DATA_TYPES_DATA_VALIDATIONS" as it contains validation functions.